### PR TITLE
Feature/cc 89 quotation page update

### PR DESCRIPTION
**Pull request by zhuocun**
image

This card contains the redesign and the responsive design for the iPad of the UI of the shopping cart.
Since I have already talked with BA about the consistency of the new UI with that on Figma, you don't need to check it :).

### CODE REVIEW

**Comment by Evezzm** [comment]
image
Will it be better if the deposit decimals can be limited to two digits as quotation?

**Comment by zhuocun** [comment]
> image
> 
> Would be better if the deposit decimals can be limited to two digits as quotation?

Fixed! Thank you :)

**Review by Evezzm — APPROVED** [review]
Well done!

**Comment by Double-Twelve** [comment]
please consider the layout of items in the shopping cart that have expired. In addition, click "All" to add it to the order. This situation should not be allowed.

image

File: src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ const NavigationBar = () => {
           </Flex>
         )}
       </Flex>
-      {!isCartOpen || !isMyTemplateOpen || !isSwitch3D ? <EditorDesignName /> : <Box></Box>}
+      {!isCartOpen && !isMyTemplateOpen ? <EditorDesignName /> : <Box></Box>}
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/NavBar/index.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/ShoppingCartContainer.tsx [hunk 2]
```diff
@@ -54,15 +54,26 @@ const ShoppingCartContainer = ({ shoppingCart }: userCartList) => {
...
         fontWeight="700"
-        color="brand.primary"
-        marginTop="23px"
-        marginBottom="20px"
+        fontStyle="normal"
+        color="#344C5C"
+        marginTop="2vh"
+        marginBottom="5vh"
+        marginLeft="auto"
+        marginRight="auto"
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/ShoppingCartContainer.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/ShoppingCartContainer.tsx [hunk 3]
```diff
@@ -81,8 +92,7 @@ const ShoppingCartContainer = ({ shoppingCart }: userCartList) => {
           </Text>
         </Flex>
       )}
-
-      <TableContainer minWidth="1080px" width="100%" overflowY="auto">
+      <TableContainer minWidth="1024px" width="90vw" overflowY="auto">
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/ShoppingCartContainer.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/ShoppingCartContainer.tsx [hunk 4]
```diff
@@ -91,42 +101,37 @@ const ShoppingCartContainer = ({ shoppingCart }: userCartList) => {
...
+                  paddingLeft="21px"
                   borderColor="#b3b2b2"
                   isChecked={allChecked}
                   isIndeterminate={isIndeterminate}
                   onChange={(e) => setCheckedItems(shoppingCart.map(() => e.target.checked))}
                 >
-                  All
+                  <Text
+                    style={columnStyle}
+                    sx={{ "text-transform": "capitalize" }}
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/ShoppingCartContainer.tsx]: If I do not use this parameter, the whole chars will be capitalized for some reason :) Fixed, thank you!

File: src/components/ShoppingCart/ShoppingCartContainer.tsx [hunk 5]
```diff
@@ -54,15 +54,26 @@ const ShoppingCartContainer = ({ shoppingCart }: userCartList) => {
     dispatch(switchOrderGeneration(true));
   };
 
+  const columnStyle = {
+    fontStyle: "normal",
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/ShoppingCartContainer.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/ShoppingCartContainer.tsx [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import { useState } from "react";
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/ShoppingCartContainer.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/CartListItem.tsx [hunk 7]
```diff
@@ -37,10 +36,16 @@ const CartListItem = ({
...
+    fontWeight: "700",
+    marginTop: "1.5vw",
+    color: "#344C5C",
+    fontStyle: "normal",
+  };
   return (
     <>
       <Tr alignItems="center" role="dataRow">
-        <Td padding="25px" sx={{ "vertical-align": "top" }} height="180px">
+        <Td sx={{ "vertical-align": "top" }}>
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/CartListItem.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/CartListItem.tsx [hunk 8]
```diff
@@ -37,10 +36,16 @@ const CartListItem = ({
     );
   };
 
+  const columnStyle = {
+    fontWeight: "700",
+    marginTop: "1.5vw",
+    color: "#344C5C",
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/CartListItem.tsx]: Fixed, thank you!

File: src/components/ShoppingCart/CartListItem.tsx [hunk 9]
```diff
@@ -54,58 +59,60 @@ const CartListItem = ({
...
           </Box>
         </Td>
+        <Td sx={{ "vertical-align": "top" }}>
+          <Text fontSize="16px" style={columnStyle}>
+            A${quotation}
+          </Text>
+        </Td>
+        <Td sx={{ "vertical-align": "top" }}>
+          <Text fontSize="16px" style={columnStyle}>
+            A${(parseFloat(quotation) * 0.02).toFixed(2)}
```
[the remark this answers is not in the document]
Comment (zhuocun) [reply src/components/ShoppingCart/CartListItem.tsx]: I'll talk to BA about it. Thank you!

**Comment by zhuocun** [comment]
> please consider the layout of items in the shopping cart that have expired. In addition, click "All" to add it to the order. This situation should not be allowed.
> 
> image

Fixed, thank you!

**Review by wwwseven — APPROVED** [review]
LGTM

**Pull request merged**